Source/Core/VideoCommon/VertexLoaderX64.cpp

```diff
@@ -119,8 +119,9 @@ int VertexLoaderX64::ReadVertex(OpArg data, VertexComponentFormat attribute, Com
     {
       CMP(32, R(count_reg), Imm8(3));
       FixupBranch dont_store = J_CC(CC_A);
-      LEA(32, scratch3, MScaled(count_reg, SCALE_4, -4));
-      MOVUPS(MPIC(VertexLoaderManager::position_cache, scratch3, SCALE_4), coords);
+      LEA(32, scratch3,
+          MScaled(count_reg, SCALE_4, -int(VertexLoaderManager::position_cache[0].size())));
```

Comment:
Shouldn't `-int(VertexLoaderManager::position_cache[0].size())` be divided by 4 here, since you're using SCALE_4 on the next line? (The problem is gone in the commit after this, which is probably why it hasn't been caught in testing)

Reply:
No, this is correct; the total offset we want is -16 bytes, and `VertexLoaderManager::position_cache[0].size()` is 4 (`sizeof(VertexLoaderManager::position_cache[0])`, which is what was used on ARM, is 16), so the `SCALE_4` below brings it to 16 correctly.

(I also tested the commit just now to make sure.)

Reply:
Oh, sizeof vs size... Yeah, makes sense then.